ubuntu

```diff
@@ -20,3 +30,46 @@ USER portage
 ENV PATH=/usr/lib/ccache:${PATH}
 WORKDIR /home/portage
 RUN pip install --user codecov coverxygen
+
+# Build cmake 3.13.4
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.13.4/cmake-3.13.4.tar.gz && \ 
+tar zxf cmake-3.13.4.tar.gz
+WORKDIR cmake-3.13.4 
+RUN sudo ./bootstrap 
+RUN sudo make -j2 && sudo make install
+WORKDIR ..
+
+# Build MSTK and Jali
+RUN wget https://github.com/MeshToolkit/mstk/archive/3.3.4.tar.gz && \
+     tar xzf 3.3.4.tar.gz
+WORKDIR MSTK-3.3.4
+RUN mkdir build
+WORKDIR build
+ENV MSTK_DIR /home/portage/MSTK-3.3.4/install 
+RUN  cmake -D CMAKE_BUILD_TYPE=Release \
+     -D CMAKE_PREFIX_PATH="/usr/include/trilinos" \
+     -D CMAKE_INSTALL_PREFIX=$MSTK_DIR \
+     -D ENABLE_PARALLEL=ON \
+     -D ENABLE_METIS=yes \
+     -D ENABLE_ZOLTAN=yes \
+     -D ENABLE_ExodusII=yes \
+     -D ENABLE_Tests=yes ..
+RUN make VERBOSE=1 -j2 && make test
+RUN make install
+RUN ls $MSTK_DIR
+
+WORKDIR ../..
+RUN git clone https://github.com/lanl/jali.git
+WORKDIR jali
+#RUN git checkout 1.0.5
```

Comment:
Not specifying a version here could be fragile going forward...

Reply:
Yes, you are right. Checking out v1.1.3 now. If this builds, we are good to go.